scripts/generateSJCL.js

```diff
@@ -12,6 +12,11 @@ async function init () {
         return
     }
 
+    if (process.platform === 'win32') {
```

Comment:
might need to improve this if we need this on windows - but shouldn't be a blocker here since the file is vendored anyway, so it would show up in CI if there were build updates etc

Reply:
Yeah so long as we don't depend on win32 to build the code I think it's fine.

Reply:
(This will break fingerprinting-canvas so if you're testing that expect it not to work well)

Reply:
but the code is vendored already, so it only breaks windows if there's a change. regardless let's change to check for 'perl' anyway